core/templates/pages/topic-viewer-page/topic-viewer-page.controller.ts

```diff
@@ -21,12 +21,12 @@ require(
   'components/common-layout-directives/common-elements/' +
   'background-banner.directive.ts');
 require('components/skills-mastery-list/skills-mastery-list.directive.ts');
+require('domain/topic_viewer/topic-viewer-backend-api.service.ts');
```

Comment:
Undo this change, (As services are imported after the directive, you can add a newline after line 29)

Reply:
ok